package/MDAnalysis/core/Selection.py

```diff
@@ -468,34 +468,6 @@ def __repr__(self):
         return "<'PointSelection' " + repr(self.cutoff) + " Ang around " + repr(self.ref) + ">"
 
 
-class CompositeSelection(Selection):
```

Comment:
Really, we never used that? It's been here since Day 1. Fair to remove it, though.